src/VisualStudio/Core/Def/Implementation/SymbolSearch/SymbolSearchContext.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.VisualStudio.Language.Intellisense.SymbolSearch;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.SymbolSearch
+{
+    internal class SymbolSearchContext : FindUsagesContext
+    {
+        internal SymbolSearchSource SymbolSource { get; }
+        internal SymbolOrigin LocalOrigin { get; }
+
+        private readonly ISymbolSearchCallback Callback;
+        public new readonly CancellationToken CancellationToken;
```

Comment:
oh goshthis is scary.  Why is this 'new'?

Reply:
It looks like this should be overriding this so the CancellationToken is available via virtual call.

(Not sure why we use that pattern, but...)